nbextensions/appCell2/widgets/tabs/resultsViewer.js

```diff
@@ -31,7 +31,8 @@ define([
             reportRendered = false,
             reportRenderTimeout = null,
             reportWidget = null,
-            reportRenderingPromise = null;
+            reportRenderingPromise = null,
+            reportParams = null;
```

Comment:
Due to how the events get passed around with scrolling, the calculated reportParams does need to be held as a part of the widget.